packages/bento-design-system/src/util/makeBentoTheme.ts

```diff
@@ -0,0 +1,13 @@
+import { deepmerge } from "deepmerge-ts";
+import { PartialDeep } from "./PartialDeep";
+import { BentoTokens } from "./bentoTokens";
+import { defaultTokens } from "./defaultTokens";
+import { createTheme } from "@vanilla-extract/css";
+import { vars } from "../vars.css";
+
+export function makeBentoTheme(tokens: PartialDeep<BentoTokens>) {
```

Comment:
With this utility, users can create new themes easily by partially overriding the bento default theme. If this utility is used, we don't have to worry about adding new breaking tokens